client/src/main/java/io/pravega/client/control/impl/Controller.java

```diff
@@ -364,6 +364,14 @@
      */
     CompletableFuture<Transaction.Status> checkTransactionStatus(final Stream stream, final UUID txId);
 
+    /**
+     * Method to get transaction list in completed state(COMMITTED, ABORTED).
+     *
+     * @param stream Stream Name
+     * @return list of transaction ids
+     */
+    AsyncIterator<io.pravega.controller.stream.api.grpc.v1.Controller.TxnResponse> listCompletedTransactions(final Stream stream);
```

Comment:
`io.pravega.controller.stream.api.grpc.v1.Controller.TxnResponse` can be imported

Reply:
Done as suggested.